src/test/scala/com/microsoft/hyperspace/index/RefreshIndexDeleteTests.scala

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
```

Comment:
Can you add a comment that this file doesn't need review?

Reply:
yeah I added one comment in the beginning saying with NOTE TO REVIEWERS which was for this exact purpose. I am not sure how many more files I will pull from the dependent PR so I did not think of adding it on per file basis

Reply:
Thank you!